Jenkinsfile

```diff
@@ -3,16 +3,12 @@
 node {
     checkout scm
 
-    nodeEnv = docker.image("node:10")
+    nodeEnv = docker.image("node:10-stretch")
```

Comment:
This moves to a Node image based on a newer version of Debian, which gives us a newer version of Git, which is in turn required by some of the commands that are run as part of the release process.